packages/wallets/src/components/Base/index.ts

```diff
@@ -13,6 +13,5 @@ export * from './WalletButton';
 export * from './WalletButtonGroup';
 export * from './WalletClipboard';
 export * from './WalletDropdown';
-export * from './WalletPasswordField';
```

Comment:
how about exporting `WalletPasswordFieldLazy` to follow the same pattern?

Reply:
hey, updated the imports/exports @matin-deriv